src/clojure_tic_tac_toe/console_ui/input_output.clj

```diff
@@ -0,0 +1,112 @@
+(ns clojure-tic-tac-toe.console_ui.input_output
+  (:require [clojure.string :as str]
+            [clojure-tic-tac-toe.utilities :refer [join-lines]]
+            [clojure-tic-tac-toe.view :as view]))
+
+(defn display-introduction []
+  (println "This is a Tic Tac Toe program.\n"))
+
+(defn display-game-mode-instructions []
+  (println (join-lines ["Please enter one of the following:"
+                        "- \"h\" to play another person"
+                        "- \"c\" to play against a computer\n"])))
+
+
+(defn- display-simple-prompt []
+  (print "> "))
+
+(defn- get-input []
+  (keyword (str/trim (read-line))))
+
+(defn get-initial-input []
+  (display-simple-prompt)
+  (flush)
+  (get-input))
+
+
+(defn- display-invalid-game-mode-message
+  [mode]
+  (printf "\n<!> Error: Game mode \"%s\" is invalid.", (name mode))
+  (printf " Must be a choice from above."))
+
+(defn- prompt-player-for-game-mode []
+  (printf "\nPlease enter the game mode: "))
+
+(defn get-game-mode
+  [mode]
+  (display-invalid-game-mode-message mode)
+  (prompt-player-for-game-mode)
+  (flush)
```

Comment:
You always call `flush` before you call `get-input`.  Maybe `flush` is in the wrong place?

Reply:
Moved to `get-input`